app/config/drupal-demo/install.sh

```diff
@@ -110,6 +110,7 @@ EOPERM
   drush -y cvapi extension.install key=org.civicrm.angularprofiles debug=1
 
   drush -y cvapi extension.install key=org.civicrm.volunteer debug=1
+  drush -y cvapi extension.install key=exportUI debug=1
```

Comment:
Also worth noting that https://github.com/colemanw/exportui/blob/master/info.xml lists the extension's name as `exportui` not `exportUI`, so this line also fails. Fixed in 5e0b0c8b2fec6aee6d70438eb3834a634a5b16ef.